tools/cldr-apps/js/src/esm/cldrInfo.mjs

```diff
@@ -150,6 +161,22 @@ function updateOpenPanelButtons() {
  * @param {Node} theObj to listen to, a.k.a. "hideIfLast"
  * @param {Function} fn the draw function
  */
+// TODO: simplify this and related code to enable a cleaner and better organized
+// appearance using modern components. The fn parameter (a.k.a. showFn) shouldn't
+// need to be preconstructed for each item in each row, attached to DOM elements,
+// and passed around as a parameter in such a complicated way.
```

Comment:
yes, it wanted to be Vue components. It's a home grown framework. All of the complexity with little of the benefit.